data/styles/clear/include/Icons.mapcss

```diff
@@ -1837,7 +1837,9 @@ line|z16-[barrier=stile],
 {icon-image: dot-m.svg;font-size: 11;}
 
 node|z16-[highway=ford],
-node|z16-[power=tower]
+node|z16-[power=tower],
+node|z16-[man_made=flagpole],
```

Comment:
`flagpole` is a more significant landmark compared to power towers and masts - let's make it more visible, e.g. z15- like a lighthouse